File: Python/NumPrimo.py
def Primo(n):
    if n < 2:
        return False
    else:
        for i in range(2,int(n**0.5)+1):
            if n % i == 0:
                return False
        return True

n = 0
if Primo(n):
    print("El numero es primo")
else:
    print("El numero no es primo")

File: C/NumPrimo.c
#include <stdio.h>
#include <math.h>

int primo(int n){

    if (n < 2){
        return 0;
    }
    
    for (int i = 2; i <= sqrt(n); i++)
    {
        if (n%i == 0){
            return 0;
        }
    }
    return 1;
}

int main(){
    int n,j = 0;

    printf("Programa que te dice los primeros n numeros par\n");
    printf("Cuantos numeros primos quieres saber?\n");
    scanf("%d",&n);
    
    printf("Lista de numeros primos: \n");
    
    for (int i = 2; j<n; i++){
        if (primo(i)){
            printf("%d\n",i);
            j++;
        }
    }
    
    return 0;
}